MarsMission/Controllers/DetailsViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  DetailsViewController.swift
+//  MarsMission
+//
+//  Created by Thanyani on 2020/11/03.
+//
+
+import UIKit
+
+class DetailsViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource{
+  
```

Comment:
Please fix the spacing between lines of code

Reply:
<img width="757" alt="Screenshot 2020-11-05 at 06 23 51" src="https://user-images.githubusercontent.com/55865505/98197510-9e9dd580-1f2f-11eb-8bdb-d5557d65c9a0.png">